app/assets/javascripts/utilities/showModal.js

```diff
@@ -1,16 +1,3 @@
-function initSignupModal() {
-  if (document.getElementById('global-signup-modal')) {
-    document.getElementsByClassName(
-      'authentication-modal__close-btn',
-    )[0].onclick = () => {
-      document.getElementById('global-signup-modal').classList.add('hidden');
-      document.body.classList.remove('modal-open');
-    };
-  }
-}
-
 function showModal(context) {
-  document.getElementById('global-signup-modal').classList.remove('hidden');
-  document.body.classList.add('modal-open');
-  initSignupModal();
+  window.presentModal('Log in to continue', '#global-signup-modal', true, 's');
```

Comment:
Refactor (blocking): The default size is `'s'`, so the last argument, `'s'`, can be omitted.

A single parameter object makes more sense here to make it clear what each argument is. (like props om a Preact component)

As well after looking through the existing code, function `showModal` is not descriptive enough. `showLoginModal` gives the full context of what that function does.

```suggestion
function showLoginModal() {
  window.presentModal({ title: 'Log in to continue', contentSelector: '#global-signup-modal', hasOverlay: true, size: 's' });  
```

Nitpick (non-blocking): instead of `window.presentModal`, consider `window.showModal` assuming the suggestion above.